shell/src/test/java/org/apache/accumulo/shell/ShellSetInstanceTest.java

```diff
@@ -119,17 +120,15 @@ public void tearDown() {
     SiteConfiguration.clearInstance();
   }
 
+  @SuppressWarnings("deprecation")
   @Test
   public void testSetInstance_Fake() throws Exception {
     ShellOptionsJC opts = createMock(ShellOptionsJC.class);
     expect(opts.isFake()).andReturn(true);
     replay(opts);
-    org.apache.accumulo.core.client.mock.MockInstance theInstance = createMock(org.apache.accumulo.core.client.mock.MockInstance.class);
-    expectNew(org.apache.accumulo.core.client.mock.MockInstance.class, "fake").andReturn(theInstance);
-    replay(theInstance, org.apache.accumulo.core.client.mock.MockInstance.class);
 
     shell.setInstance(opts);
-    verify(theInstance, org.apache.accumulo.core.client.mock.MockInstance.class);
+    Assert.assertTrue(shell.getInstance() instanceof org.apache.accumulo.core.client.mock.MockInstance);
```

Comment:
For some reason the power mock stuff was not working, so I went with this approach which seemed simpler